lib/ansible/modules/network/enos/enos_config.py

```diff
@@ -248,32 +243,35 @@ def main():
         after=dict(type='list'),
 
         match=dict(default='line', choices=['line', 'strict', 'exact', 'none']),
-        replace=dict(default='line', choices=['line', 'block', 'config']),
+        replace=dict(default='line', choices=['line', 'block']),
 
         config=dict(),
+        defaults=dict(type='bool', default=False),
+        passwords=dict(type='bool', default=False),
```

Comment:
`DOCUMENTATION:` for `passwords:` needs adding.
From the name `passwords:` it isn't clear to me what this is doing. Is there a better name?

Reply:
Reverting the changes back.

Reply:
Reverting the changes back.